src/assets/src/components/queue.tsx

```diff
@@ -175,14 +175,27 @@ function QueueAttendingJoined(props: QueueAttendingProps) {
         ? <small className="card-text-spacing meeting-type-message">A Host has been assigned to this meeting. Meeting Type can no longer be changed.</small>
         : <button disabled={props.disabled} onClick={props.onShowDialog} type="button" className="btn btn-link">Change</button>;
     const agendaText = props.queue.my_meeting!.agenda
+    const numberInLine = props.queue.my_meeting!.line_place + 1;
+    const notificationBlurb = numberInLine > 1
+        && (
+            <Card.Text>
+                <Alert variant="info">
+                    Did you know? You can receive a SMS (text) message when 
```

Comment:
I think this should be "You can receive **an** SMS (text) message", since SMS phonetically starts with a vowel sound of "es". See https://www.merriam-webster.com/words-at-play/is-it-a-or-an

Reply:
Also, we may want to consider wrapping the text here with `small`, since the Card text generally is kind of large. That would make this text the same size as the instructional text for the agenda field.